### ISSUE TITLE
New feature question : homepage button in projects

### ISSUE THREAD

**Issue by lucitemple** [hashirshoaeb/home#79]
Hi @hashirshoaeb 

I've forked your project and am working on adding a homepage button on the project card. You can see what I've done so far [here](https://github.com/lucitemple/home/blob/feature-homepage-button/src/components/home/ProjectCard.jsx).

What currently works:
- button displays correctly next to other buttons
- the github pages url address has passed down correctly to the buttons component as props

What isn't working:
1) In browser, the url on the button seems to be undefined, and if clicked it opens a new page with an error:

```
TypeError: cannot read property of offsetTop of undefined. 
src/components/Navbar.jsx:29
```
2) Sometimes the project content won't load, the dummy/ null does instead, with these error messages in console:

```
api.github.com/users/lucitemple/repos?sort=updated&direction=desc:1 
Failed to load resource: the server responded with a status of 403 ()
```
```
index.js:1 Request failed with status code 403
console.<computed> @ index.js:1
```

Do you have any idea what is going on and how I can fix it?

Thanks!

**Comment by hashirshoaeb** [comment]
Hi @lucitemple  
I'm sorry to respond late.  I have really tight schedule these days. Will you mind reminding me next weekend? I'll be happy to help. 

@telriot can you assist us on this issue please?

Best,

**Comment by lucitemple** [comment]
@hashirshoaeb Thanks, there's no hurry, I'll check in with you on the weekend to see if you're able to help then :)

**Comment by lucitemple** [comment]
Hi @hashirshoaeb, just checking back in with you in case you have some time this weekend to take a look at this. No worries if you are too busy, I don't want to intrude.

**Comment by hashirshoaeb** [comment]
Hi @lucitemple I have checked your code that was in your, `feature-homepage-button` branch.

On running I saw the `homepage` button that you have added. 
<img src="https://user-images.githubusercontent.com/35165481/117568963-b02d8900-b0dc-11eb-9747-f7e75a8f5266.png" width=350 />

On clicking it, nothing was happening. So I started debugging it and found a bug [here](https://github.com/lucitemple/home/blob/ad31cf0e881a54bbcd3e35fbd807c730fcc0c930/src/components/home/ProjectCard.jsx#L50).
```js
const CardButtons = ({ svn_url }, { homepage }) => {
```
`homepage` button was undefined. So following is the quick fix:
```js
const CardButtons = ({ svn_url , homepage }) => {
```

Thanks for your patience. 
Best,
Hashir.
